examples/mnist/mnist_lib.py

```diff
@@ -173,39 +146,39 @@ def get_datasets():
   return train_ds, test_ds
```

Comment:
Re the line above, `tfds.builder('mnist')`: Looks like we're hard-coding the MNIST dataset version in the tfds JSON file we have committed in this PR, but this code here doesn't specify a version. We should change the call to `tfds.builder` to have an explicit version of the MNIST TFDS dataset.

Moreover, we should maybe try to find a way to guarantee that nobody can accidentally bump this version without also updating the JSON file

Reply:
@mohitreddy1996 presuaded me that this should already break because `tfds.mock_data` wouldn't be able to find the JSON file otherwise. Worth doing a quick check by setting version explicitly here and seeing the test fail.